### DISCUSSION TITLE
Not necessarily an issue but a request (mirror sites)

### DISCUSSION THREAD

**Discussion by Alharic** [nfriedly/nodeunblocker.com#37] [Ideas]
Is there a way to make a list of all versions of this site and all mirrors deployed so that if one stops working there are always more (only cyclic works on my computer so I can only make one at a time) I can open sites from Heroku, railway, and IBM and a lot more I just cant deploy them. thank you.

**Comment by nfriedly** [comment]
It's not really possible to make a list of _all_ versions & mirrors, but you're welcome to make a list of some of them. I don't really keep track of it any more, but I know there's a discord channel, https://discord.gg/unblock, where some folks do things like that.

**Reply by Alharic** [reply]
all right, thank you for the reply. I will now go on a hunt.

**Reply by nfriedly** [reply]
Op, I think I got interrupted when I was writing that. I just now edited it to include https://discord.gg/unblock
